ibis/backends/bigquery/tests/test_compiler.py

```diff
@@ -186,7 +186,7 @@ def test_day_of_week(case, expected, dtype, strftime_func):
 )
 def test_literal_timestamp_or_time(case, expected, dtype):
     expr = ibis.literal(case, type=dtype).hour()
-    result = ibis.bigquery.compile(expr)
+    result = ibis.backends.bigquery.compile(expr)
```

Comment:
This and the other 7 similar changes in this file also need to be reverted.

Reply:
done!